dashboard/installation/dashboard-migration.md

```diff
@@ -41,7 +52,20 @@ The Invictus V2 release pipeline will create a new App Service dashboard resourc
 
 **NOTE: By doing this you will lose access to the old dashboard and the ability to view and query its historic data.**
 
+## Add Role Assignments
+
+In addition to the flowhandlerjob function, in V2 also the dashboardgateway must have Logic App Contribute access on the resource group or subscription which contains the Logic Apps.
+See also [Access Control Rights](../accesscontrolrights.md)
+
+## Clear migrated AzureAD users from CosmosDB container
+After migration, you must delete all users with filter {"IsAadUser" : true} from the Users collection in the CosmosDB database.
+These users will be recreated at first login with the correct UserId.
+
 ## Common Issues When Migrating
 - `You cannot change the OS hosting your app at this time. Please recreate your app with the desired OS.`
 
   If you are passing the `servicePlanName` or `autoscaleForPlanName` (or both) parameters to the dashboard release pipeline, these must be updated by adding `-linux` to the end of their values (feel free to maintain your own naming conventions). *This change is required for the dashboard pipeline only*.
+
+## Invictus artifacts deployment
```

Comment:
I think this sub-title could be improved. Maybe something like "Invictus Configure Dashboard Script Update"